lib/help.js

```diff
@@ -0,0 +1,141 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: generator-loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+var g = require('./globalize');
+var _ = require('lodash');
+var table = require('text-table');
+var helpers = require('./helpers.js');
+
+/**
+ * Customize the help message based on how the generator is invoked (slc|yo)
+ * @param {Object} generator The generator instance
+ * @param {String} cmd The cmd name
+ * @returns {String} The customized help string
+ */
+exports.customHelp = function(generator, usageFilePath, cmd) {
+  var command = cmd || helpers.getCommandName();
+
+  // Build usage
+  var helpStr = [
+    g.f('Usage:'),
+    ' ' + usageBuild(generator),
+    '',
+  ];
+
+  // Build options
+  if (Object.keys(generator._options).length) {
+    helpStr = helpStr.concat([
+      g.f('Options:'),
+      optionsBuild(generator),
+      '',
+    ]);
+  }
+
+  // Build arguments
+  if (generator._arguments.length) {
+    helpStr = helpStr.concat([
+      g.f('Arguments:'),
+      argumentsBuild(generator),
+      '',
+    ]);
+  }
+
+  // Append USAGE file if any
+  var exists = g.f(usageFilePath);
```

Comment:
@bajtos strong-globalize returns the globalized help files as long as the file is in `intl/en`.
See example https://github.com/strongloop/strong-globalize#help-txt-files